### PR TITLE
Feature/challenge

### PR DESCRIPTION
**Pull request by tkmct**

### CODE REVIEW

File: packages/plasma-light-client/src/LightClient.ts [hunk 1]
```diff
@@ -593,7 +606,76 @@ export default class LightClient {
...
+        }
+      }
+    )
+
+    this.adjudicationContract.subscribeNewPropertyClaimed(
+      async (gameId, property, createdBlock) => {
+        console.log('property is claimed', gameId, property, createdBlock)
+        if (
+          property.deciderAddress.data ===
+          this.deciderManager.getDeciderAddress('Exit').data
```
Comment (syuhei176) [inline packages/plasma-light-client/src/LightClient.ts]: We want equals method for primitives.

File: packages/plasma-light-client/src/LightClient.ts [hunk 2]
```diff
@@ -593,7 +606,76 @@ export default class LightClient {
...
+            if (!decision.outcome && decision.challenges.length > 0) {
+              const challenge = decision.challenges[0]
+              const challengingGameId = Keccak256.hash(
+                ovmContext.coder.encode(challenge.property.toStruct())
+              )
+              this.adjudicationContract.challenge(
+                gameId,
+                challenge.challengeInput
+                  ? List.from(Bytes, [challenge.challengeInput])
+                  : List.from(Bytes, []),
```
Comment (syuhei176) [inline packages/plasma-light-client/src/LightClient.ts]: ideally, `challenge.challengeInputs` or `challenges.map(c => c.challengeInputs)` here?

File: packages/plasma-light-client/src/LightClient.ts (hunk 1 above)
Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts]: true. Let's create an issue

File: packages/plasma-light-client/src/LightClient.ts (hunk 2 above)
Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts]: idealy, `challenge.challengeInput`

**Pull request merged**